packages/dapp-svelte-wallet/ui/lib/ListCard.svelte

```diff
@@ -11,14 +11,19 @@
 
   export let expandIcon = 'arrow_right';
 
-  let expanded = [];
+  // true iff new items should start expanded
+  export let expandDefault = false;
 
-  const toggle = item => {
-    // console.log('toggle', item);
-    if (expanded.includes(item)) {
-      expanded = expanded.filter(it => item !== it);
+  let expandState = [];
+
+  $: isExpanded = id => expandState.includes(id) !== expandDefault;
```

Comment:
I don't think I've seen this notation. Is this from Svelte? What does it do?

Reply:
In Svelte, a statement that is labelled with `$:` means "rerun it when the inputs change".